docs/src/main/paradox/project/migration-guides.md

```diff
@@ -1,14 +1,12 @@
 ---
-project.description: Akka version migration guides.
+project.description: Pekko version migration guides.
 ---
 # Migration Guides
 
 @@toc { depth=1 }
 
 @@@ index
 
-* [migration-guide-2.5.x-2.6.x](migration-guide-2.5.x-2.6.x.md)
-* [migration-guide-2.4.x-2.5.x](migration-guide-2.4.x-2.5.x.md)
-* [migration-guide-old](migration-guide-old.md)
+TODO ADD AKKA -> PEKKO MIGRATION GUIDE HERE
```

Comment:
it would probably be interesting to elaborate a migration guide from Akka to Pekko at some point

Reply:
Could you include a link to the real Akka migration guides and encourage users who are not using versions of Akka older than v2.6.x to start by reading how to migrate to v2.6.x? They can assume that they'll need to allow for those migration steps as well as the Akka to Pekko migration steps (when we provide those).

Reply:
yes good idea, done